playbooks/reboot_cluster.yml

```diff
@@ -46,3 +54,58 @@
     service:
       name: mysql
       state: started
+
+- hosts: compute:!controller
```

Comment:
I can't comment, but on the plays above this there is a reboot performed on `db` nodes, which are the controller nodes, which means those nodes would reboot twice.